modules/id5IdSystem.js

```diff
@@ -25,6 +25,19 @@ export const id5IdSubmodule = {
   decode(value) {
     return (value && typeof value['ID5ID'] === 'string') ? { 'id5id': value['ID5ID'] } : undefined;
   },
+
+  // todo: who can confirm the format?
```

Comment:
other than the question below on value in the `source` field, i can confirm the format is correct. @padurgeat fyi